include/spirv-tools/libspirv.hpp

```diff
@@ -111,6 +117,9 @@ class SpirvTools {
   // Like the previous overload, but takes an options object.
   bool Validate(const uint32_t* binary, size_t binary_size, const ValidatorOptions& options) const;
 
+  bool AggregateStats(const uint32_t* binary, size_t binary_size,
```

Comment:
But more importantly, why is this in the public header when you can't get any info out of it?  In fact you can't even call it if you just have the public header, because you can't form a valid non-null pointer to a SpirvStats object.

You should move this whole entry point inside of a header under source.  It's not something external code can use.